src/lib/broadcast.jl

```diff
@@ -106,10 +100,10 @@ end
 end
 
 @adjoint broadcasted(::typeof(conj), x::Numeric) =
-  conj.(x), z̄ -> (nothing, conj.(z̄))
+  conj(x), z̄ -> (nothing, conj(z̄))
```

Comment:
This ought to save a copy, when not needed, since
```
julia> @btime conj.($(rand(10^6)));
  285.667 μs (2 allocations: 7.63 MiB)

julia> @btime conj($(rand(10^6)));
  1.500 ns (0 allocations: 0 bytes)
```
And since arrays are being regarded as immutable, it's safe to remove trivial copies.